### PR TITLE
[marathon-lb] Version bump to 1.1.4.

### PR DESCRIPTION
**Pull request by brndnmtthws**
It works:

screen shot 2016-04-04 at 4 42 15 pm
screen shot 2016-04-04 at 4 42 27 pm

### CODE REVIEW

File: repo/packages/M/marathon-lb/2/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  ],
+  "upgradeStrategy": {
+    "minimumHealthCapacity": {{marathon-lb.minimumHealthCapacity}},
+    "maximumOverCapacity": {{marathon-lb.maximumOverCapacity}}
+  },
+  "args":[
+    "sse",
+    "-m", "{{marathon-lb.marathon-uri}}",
+    "--health-check",
+  {{^marathon-lb.bind-http-https}}
```
Comment (jdef) [inline repo/packages/M/marathon-lb/2/marathon.json.mustache]: what's the difference between using the carrot symbol vs pound?

Comment (jdef) [inline repo/packages/M/marathon-lb/2/marathon.json.mustache]: "if this is not defined, then..."?

Comment (brndnmtthws) [inline repo/packages/M/marathon-lb/2/marathon.json.mustache]: That's correct. If `marathon-lb.bind-http-https` isn't defined, show this code.

**Comment by jsancio** [comment]
@brndnmtthws Minor comment but see #354. I fixing pre-existing packages but we should make sure that new packages have those changes. Let me know if you have any questions.

**Comment by brndnmtthws** [comment]
Updated with @jsancio's changes from #354. Ready to go.

**Pull request merged**